README.md

```diff
@@ -1,24 +1,26 @@
 # Gateway API Inference Extension 
 
-The Gateway API Inference Extension came out of [wg-serving](https://github.com/kubernetes/community/tree/master/wg-serving) and is sponsored by [SIG Network](https://github.com/kubernetes/community/blob/master/sig-network/README.md#gateway-api-inference-extension). This repo contains: the load balancing algorithm, [ext-proc](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter) code, CRDs, and controllers of the extension.
+The Gateway API Inference Extension - also known as an inference gateway - improves the tail latency and throughput of OpenAI completion requests when load balancing a group of LLM servers on Kubernetes with kv-cache awareness. It provides Kubernetes-native declarative APIs to route client model names to use-case specific LoRA adapters and control incremental rollout of new adapter versions, A/B traffic splitting, and safe blue-green base model and model server upgrades. By adding operational guardrails like priority and fairness to different client model names, the inference gateway allows a platform team to safely serve many different GenAI workloads on the same pool of shared foundation model servers for higher utilization and fewer required accelerators.
```

Comment:
s/with kv-cache awareness/with model server load awareness (kv-cache and queue length)